### ISSUE TITLE
Turning adb on twice in apps/tools menu crashes system

### ISSUE THREAD

**Issue by mennoliefstingh** [rg35xx-cfw/Koriki#97]
### What happened?

The new bash scripts for turning adb on/off are great, but executing `adb_on.sh` twice results in either a 10-15s long 'respring' or a full power cycle of the system.

**Comment by acmeplus** [comment]
Yeah, those were some quick scripts to get it to work. I will add some checks to avoid that.
